schemas/camera_x_rays/dd_camera_x_rays.xsd

```diff
@@ -77,6 +77,15 @@
 						</xs:appinfo>
 					</xs:annotation>
 				</xs:element>
+				<xs:element name="t_e_magnetic_axis" type="signal_flt_1d_validity">
```

Comment:
I'm happy to approve this PR but note we're not always fully consistent with our naming.  For instance, we have `psi_axis` and `q_axis` in the equilibrium IDS, not `psi_magnetic_axis` and `q_magnetic_axis`.

Reply:
Since explicit is better than implicit, I think we should keep what you've put here.